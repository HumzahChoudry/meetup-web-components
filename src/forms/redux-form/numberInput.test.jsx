import React from 'react';
import { shallow } from 'enzyme';
import ReduxFormNumberInput from './NumberInput';

describe('redux-form NumberInput', function() {
	// props structured to match what redux-form provides to <Field> `component`
	const props = {
		input: {
			label: 'Number of internets',
			name: 'internets',
			value: 10,
		},
	};

	it('renders a NumberInput component with expected attributes from mock data', () => {
		const component = shallow(<ReduxFormNumberInput {...props} />);
		expect(component).toMatchSnapshot();
	});
});
