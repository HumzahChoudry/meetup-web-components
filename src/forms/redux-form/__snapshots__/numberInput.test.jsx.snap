// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`redux-form NumberInput renders a NumberInput component with expected attributes from mock data 1`] = `
<WithErrorList(NumberInput)
  label="Number of internets"
  name="internets"
  onChange={[Function]}
  value={10}
/>
`;
